File: Python/program11.py
# program11.py
def addition_two_numbers(f_no1: float, f_no2: float) -> float:
    # If the input is negative then convert it into positive
    if f_no1 < 0.0:
        f_no1 = -f_no1
        
    if f_no2 < 0.0:
        f_no2 = -f_no2
        
    return f_no1 + f_no2

def main():
    try:
        val1 = float(input("Enter first number : "))
        val2 = float(input("Enter second number : "))
        
        result = addition_two_numbers(val1, val2)
        print(f"Addition is : {result}")
    except ValueError:
        print("Invalid input. Please enter numbers.")

if __name__ == "__main__":
    main()

File: Go/program11.go
/*
Package main
Description: Accepts two floats, converts any negative input to positive,

	and then performs addition. (|a| + |b|)

Date: May 12, 2026
*/
package main

import (
	"fmt"
	"math"
)

// AdditionOfAbsolutes converts both inputs to their absolute
// values before adding them together.
func AdditionOfAbsolutes(nos1 float64, nos2 float64) float64 {
	// Convert inputs to absolute values first
	val1 := math.Abs(nos1)
	val2 := math.Abs(nos2)

	return val1 + val2
}

func main() {
	var nos1, nos2 float64

	fmt.Println("--- Addition of Absolute Inputs ---")

	fmt.Print("Enter first number: ")
	fmt.Scan(&nos1)

	fmt.Print("Enter second number: ")
	fmt.Scan(&nos2)

	result := AdditionOfAbsolutes(nos1, nos2)

	fmt.Printf("\nProcessing: |%v| + |%v|\n", nos1, nos2)
	fmt.Printf("Final Result: %v\n", result)
}
